src/function/pragma/pragma_functions.cpp

```diff
@@ -277,6 +277,20 @@ static void PragmaDebugManyFreeListBlocks(ClientContext &context, const Function
 	config.debug_many_free_list_blocks = true;
 }
 
+static void PragmaDebugWindowMode(ClientContext &context, const FunctionParameters &parameters) {
```

Comment:
Could you add some tests where this is used, perhaps run some window functions in a loop toggling each of these different modes?

Reply:
Good point. And I think that should be part of the test suite for such functions to make sure they all give the same answer!